core/store/config/general_config.go

```diff
@@ -294,10 +290,8 @@ func (c *generalConfig) Validate() error {
 		return errors.Errorf("P2P_ANNOUNCE_PORT was given as %v but P2P_ANNOUNCE_IP was unset. You must also set P2P_ANNOUNCE_IP if P2P_ANNOUNCE_PORT is set", c.P2PAnnouncePort())
 	}
 
-	// TODO: Remove when implementing
-	// https://app.clubhouse.io/chainlinklabs/story/8096/fully-deprecate-minimum-contract-payment
-	if c.viper.IsSet("MINIMUM_CONTRACT_PAYMENT") {
-		logger.Warn("MINIMUM_CONTRACT_PAYMENT is now deprecated and will be removed from a future release, use MINIMUM_CONTRACT_PAYMENT_LINK_JUELS instead.")
+	if _, exists := os.LookupEnv("MINIMUM_CONTRACT_PAYMENT"); exists {
```

Comment:
Might be better to put it in the `init` somewhere and just exit the application if it exists